config/default.js

```diff
@@ -383,7 +383,7 @@ module.exports = {
   enableFeatureExperienceSurvey: false,
   dismissedExperienceSurveyCookieName: 'dismissedExperienceSurvey',
 
-  enableFeaturePromotedShelf: false,
+  enableFeatureSponsoredShelf: true,
 
```

Comment:
so, just to confirm, there are no configs where we want `enableFeaturePromotedShelf: false`?  (Because those would previously be inheriting from `default.js` where it was already `false` so wouldn't have been declared.)

Reply:
Correct!